README.md

```diff
@@ -88,38 +101,41 @@ To create service credentials, access to Event Notifications and access to call
+| <a name="input_cbr_rules"></a> [cbr\_rules](#input\_cbr\_rules) | The optional list of context-based restrictions (CBR) rules to create. | <pre>list(object({<br>    description = string<br>    account_id  = string<br>    rule_contexts = list(object({<br>      attributes = optional(list(object({<br>        name  = string<br>        value = string<br>    }))) }))<br>    enforcement_mode = string<br>  }))</pre> | `[]` | no |
+| <a name="input_existing_kms_instance_crn"></a> [existing\_kms\_instance\_crn](#input\_existing\_kms\_instance\_crn) | "The Cloud Resource Name (CRN) of the Hyper Protect Crypto Services (HPCS) or Key Protect instance. This input is required only if `var.kms\_encryption\_enabled` is set to true. | `string` | `null` | no |
+| <a name="input_kms_encryption_enabled"></a> [kms\_encryption\_enabled](#input\_kms\_encryption\_enabled) | Set this input value to `true` to control the encryption keys that are used to encrypt the data that you store in the Event Notifications instance. If this value is set to `false`, the data is encrypted by using randomly generated keys. For more information, see [Managing encryption](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption) | `bool` | `false` | no |
+| <a name="input_kms_endpoint_url"></a> [kms\_endpoint\_url](#input\_kms\_endpoint\_url) | The URL of the key management service (KMS) endpoint to use when configuring KMS encryption. The Hyper Protect Crypto Services (HPCS) endpoint URL format is `https://api.private.<REGION>.hs-crypto.cloud.ibm.com:<port>` and the Key Protect (KP) endpoint URL format is `https://<REGION>.kms.cloud.ibm.com` | `string` | `null` | no |
+| <a name="input_name"></a> [name](#input\_name) | The name of the Event Notifications instance that is created by this module. | `string` | n/a | yes |
+| <a name="input_plan"></a> [plan](#input\_plan) | The pricing plan of the Event Notifications instance. The following pricing plans are available: `Lite`, `Standard` | `string` | `"standard"` | no |
+| <a name="input_region"></a> [region](#input\_region) | The IBM Cloud region where the Event Notifications resource is created. The following regions are supported: `us-south` (Dallas), `eu-gb` (London), `eu-de` (Frankfurt), `au-syd` (Sydney), `eu-es` (Madrid) | `string` | `"us-south"` | no |
+| <a name="input_resource_group_id"></a> [resource\_group\_id](#input\_resource\_group\_id) | The ID of the resource group where the Event Notifications instance is created. | `string` | n/a | yes |
+| <a name="input_root_key_id"></a> [root\_key\_id](#input\_root\_key\_id) | The key ID of a root key, existing in the key management service (KMS) instance that is passed in `var.existing_kms_instance_crn`, which is used to encrypt the data encryption keys (DEKs) which are then used to encrypt the data. This variable is required if `var.kms_encryption_enabled` is set to `true`. | `string` | `null` | no |
```

Comment:
I think `Required if...` is ok. But make sure that they enclose the var names in backticks 

```md
Required if the `key_encryption_enable` input is set to `true`.
```